tests/basic/telemetry_test.go

```diff
@@ -88,10 +88,9 @@ func IsPKS() bool {
 	if isOpBased {
 		log.Info("This is PX Operator based install, checking if StorageCluster has PKS annotation set to true...")
 		spec, err := Inst().V.GetDriver()
-		Expect(err).ToNot(HaveOccurred())
+		log.FailOnError(err, "Failed to get StorageCluster object")
 		isPks, err := strconv.ParseBool(spec.Annotations["portworx.io/is-pks"])
```

Comment:
can we change it like this 

val, ok:= spec.Annotations["portworx.io/is-pks"]
if ok {
isPks, err := strconv.ParseBool(val)
log.FailOnError(err, "Failed to check PKS")
return isPKS
}

Reply:
This `strconv.ParseBool(spec.Annotations["portworx.io/is-pks"])` already covers that fact that this annotation exists, if it doesn't, I am assuming its not PKS.
Basically, if this annotation doesn't exists or its value is not `true` or `false`, PX itself would fail to deploy as it needs this annotation to deploy properly on PKS.

Reply:
ya but for non PKS set up this annotation will be not there and we will get error while parsing to bool. since here anyway you are checking for err == nil, it should be fine